Source/WebCore/platform/graphics/Damage.h

```diff
@@ -91,82 +105,96 @@ class Damage {
             return;
 
         if (rect.contains(m_minimumBoundingRectangle)) {
-            m_rects = { rect };
+            m_rects.clear();
+            m_rects.append(rect);
```

Comment:
`m_rects = { rect };` seemed clear enough, out of curiosity, is there a reason you prefer clear + append?

Reply:
Yes, maybe this is already optimized by the compiler, but since the Vector has one stack allocated item, in this case clearing the existing one and adding the rect seems cheaper than creating a new instance.